todo_app/tests/test_view_model.py

```diff
@@ -76,22 +76,25 @@ def test_older_done_items(create_items):
 # def test_index_page(monkeypatch, client):
```

Comment:
Not particularly relevant here but all this commented out code is very ugly - worth noting that we can tag them with `@pytest.mark.skip()` to temporarily disable them in a slightly more visible way (each run will note that it was skipped, which is more likely to prompt someone to update them than simply commenting them out!)